components/user_settings/import_theme_modal.tsx

```diff
@@ -64,15 +64,36 @@ class ImportThemeModal extends React.PureComponent<WrappedComponentProps, State>
             return;
         }
 
+        /*
+         * index mapping of slack theme format (index => slack-property name)
+         *
+         * |-------|-------------------------|-------------------------|
+         * | index | Slack theme-property    | MM theme-property       |
+         * |-------|-------------------------|-------------------------|
+         * |   0   | Column BG               | sidebarBg               |
+         * |   1   | ???                     | sidebarHeaderBg         |
+         * |   2   | Active Item BG          | sidebarTextActiveBorder |
+         * |   3   | Active Item Text        | sidebarTextActiveColor  |
+         * |   4   | Hover Item BG           | sidebarTextHoverBg      |
+         * |   5   | Text Color              | sidebarText             |
+         * |   6   | Active Presence         | onlineIndicator         |
+         * |   7   | Mention Badge           | mentionBg               |
+         * |   8   | TOP-NAV BG              | --- (desktop only)      |
+         * |   9   | TOP-NAV Text            | --- (desktop only)      |
+         * |-------|-------------------------|-------------------------|
+         *
+         * values at index 8 + 9 are only for the desktop app
+         */
+
         const [
-            sidebarBg, // 0
-            sidebarHeaderBg, // 1
-            sidebarTextActiveBorder, // 2
-            sidebarTextActiveColor, // 3
-            sidebarTextHoverBg, // 4
-            sidebarText, // 5
-            onlineIndicator, // 6
-            mentionBg, // 7
+            sidebarBg,
```

Comment:
Is there ever a case where one of these values is missing?

Reply:
no, that cannot happen, since we check for the length of the array before ever coming to this step!